firefly_client/firefly_client.py

```diff
@@ -755,6 +761,10 @@ def fetch_table(self, file_on_server, tbl_id=None, page_size=1):
             A table ID. It will be created automatically if not specified.
         page_size : `int`, optional
             The number of rows to fetch.
+        table_index : `int`, optional
+            The table to be fetched in case `file_on_server` contains multiple tables. It is extension number for
+            a fits file or table index for a votable file. In default, table of extension 1 from a fits file or
```

Comment:
"a FITS file or the table index for a VOTable file. If unspecified, the server will fetch extension 1 from a FITS file or"